webpack/assets/stylesheets/application/partials/_charts.scss

```diff
@@ -131,7 +131,7 @@
 }
 
 .statistics-pie.small .overlay {
-  background: image-url('pie_overlay.png') no-repeat;
+  background: url('#{$local-img-path}/pie_overlay.png') no-repeat;
```

Comment:
I believe this is not in used anymore. I didn't find a code reference to the overlay css class.